test/eth.accounts.signTransaction.js

```diff
@@ -453,10 +453,9 @@ describe("eth", function () {
                     var testAccount = ethAccounts.privateKeyToAccount(test.privateKey);
                     assert.equal(testAccount.address, test.address);
 
-                    testAccount.signTransaction(test.transaction).then(function (tx) {
-                        assert.equal(tx.messageHash, test.messageHash);
-                        assert.equal(tx.transactionHash, test.transactionHash);
-                        assert.equal(tx.rawTransaction, test.rawTransaction);
```

Comment:
All these assertions should remain if this change is to be non-breaking.

Reply:
Unresolving for the moment - these text changes are an example of how the PR breaks the current return object's interface.